src/components/task-plan/builder.cjsx

```diff
@@ -377,9 +369,10 @@ module.exports = React.createClass
 
   renderEnabledTasking: (plan) ->
     {taskingOpensAt, taskingDueAt} = @getDefaultPlanDates(plan.id)
-    maxOpensAt = new moment(TaskPlanStore.getDueAt(@props.id, plan.id)).subtract(1, 'day').toDate()
+    maxOpensAt = TimeHelper.makeMoment(TaskPlanStore.getDueAt(@props.id, plan.id), ISO_DATE_FORMAT).subtract(1, 'day')
+    minDueAt = TimeHelper.makeMoment(TaskPlanStore.getMinDueAt(@props.id, plan.id), ISO_DATE_FORMAT)
 
-    <BS.Row key={plan.id} className="tasking-plan tutor-date-input">
+    <BS.Row key="tasking-enabled-#{plan.id}" className="tasking-plan tutor-date-input">
```

Comment:
This was just to make it more clear, I thought there was a bug due to this, but there wasn't.